scripts/benchmarking/benchmarking_linux.sh

```diff
@@ -0,0 +1,190 @@
+
```

Comment:
I'd drop benchmarking from the file names since they are already in a folder named benchmarking